modules/platforms/cpp/DEVNOTES.md

```diff
@@ -0,0 +1,44 @@
+## Prerequisites
+* C++ compiler supporting C++17
+* Java 11 SDK
+* Maven 3.6.0+ (for building)
+* Conan C/C++ package manager
+* CMake 3.10+
+
+## Build Java
+In repo root: `mvn clean install -DskipTests`
+
+## Build C++
+
+### For Developers
+Building in debug mode with tests. In this dir:
+```bash
+mkdir cmake-build-debug
+cd cmake-build-debug
+conan install .. -s build_type=Debug
+cmake .. -DENABLE_TESTS=ON
```

Comment:
This fails on my machine with `Could NOT find Boost (missing: Boost_INCLUDE_DIR unit_test_framework)`, even though Boost is installed from Conan. Something else might be missing.

Reply:
Fixed, please re-check.

Reply:
Works now, thanks!